src/js/containers/Institutions.jsx

```diff
@@ -21,8 +22,11 @@ export class InstitutionContainer extends Component {
 
 export function mapStateToProps(state) {
   const { institutions } = state.app.institutions
+  const { submission, error, filingPeriod } = state.app
+  const unsortedFilings = state.app.filings
 
-  const { filings, submission, error, filingPeriod } = state.app
+  let filings = unsortedFilings
+  filings.filings = unsortedFilings.filings.sort(sortFilings)
```

Comment:
I don't think we should mutate the state here, was the sort not working in the `receiveInstitutions` action creator?

Reply:
👍  I'll keep you posted.